### PR TITLE
RenderException Handling

### PR DESCRIPTION
**Pull request by pltaylor**
Handle RenderExceptions in a way that can be accessed easily.  Addresses #542

### CODE REVIEW

File: Source/OxyPlot/PlotModel/PlotModel.Rendering.cs [hunk 1]
```diff
@@ -127,6 +127,7 @@ protected virtual void RenderOverride(IRenderContext rc, double width, double he
                             "An exception of type {0} was thrown when rendering the plot model.\r\n{1}",
                             exception.GetType(),
                             exception.GetBaseException().StackTrace);
+                    this.updateException = exception;
```
Comment (objorke) [inline Source/OxyPlot/PlotModel/PlotModel.Rendering.cs]: Wouldn't it better to create a new field `renderException`? And a `GetLastRenderException()` method? Or rename to `lastException` and `GetLastException()`?

**Comment by objorke** [comment]
Please also update the change log with a reference to the issue. :-)

File: Source/OxyPlot/PlotModel/PlotModel.Rendering.cs (hunk 1 above)
Comment (pltaylor) [inline Source/OxyPlot/PlotModel/PlotModel.Rendering.cs]: I will glad to do either.  Which way would you prefer?

**Comment by pltaylor** [comment]
This changelog or is there one in code that I should reference?

**Comment by svendu** [comment]
The file to update, is called CHANGELOG.md. You should find it in the root of your oxyplot sources

**Pull request merged**